build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'groovy'
-    id 'fabric-loom' version '1.8-SNAPSHOT'
+    id 'fabric-loom' version '1.7-SNAPSHOT'
```

Comment:
🤔

Reply:
depends on which commits are selected it seems, it does show at that link, just not on [this link](https://github.com/John-Paul-R/Essential-Commands/pull/305/files), probably because the comment is on a line that doesn't exist anymore